source/docs/software/advanced-control/state-space/state-space-drivetrain-controllers.rst

```diff
@@ -0,0 +1,48 @@
+State-space Differential Drive Controllers
+==========================================
+
+WPILib includes two new path following controls for differential drivetrains: the ``LTVUnicycleController`` and ``LTVDiffDriveController``. These controllers can be used to accurately track trajectories and account for external disturbances.
+
+Linear Time-Varying Unicycle Controller
+---------------------------------------
+
+Similar to Ramsete, the LTV (Linear Time-Varying) Unicycle Controller generates ``ChassisSpeed`` (C++/Java) references to steer the robot along a trajectory. The system used by this controller has states :math:`\begin{bmatrix}x & y & \theta \end{bmatrix}^T` in the global frame, and linear and angular velocity inputs :math:`\begin{bmatrix}v & \omega \end{bmatrix}^T`. However, this controller utilizes tolerances grounded in reality to pick gains rather than the magical Beta and Zeta gains used by Ramsete.
+
+The LTV Unicycle controller generates feedback gains using a Linear Quadratic Regulator to balance control effort and state excursion. Because the system has states :math:`\begin{bmatrix}x & y & \theta \end{bmatrix}^T` and inputs :math:`\begin{bmatrix}v & \omega \end{bmatrix}^T`, the LQR used to compute the controller's feedback gain takes maximum desired error tolerances for :math:`x`, :math:`y` and :math:`\theta` (in meters and radians) and maximum desired control effort in meters per second and radians per second. Note that because the controller deals with cross-track error, a tight heading tolerance will penalize steering the robot back toward the trajectory. Furthermore, the control effort tolerances are the maximum effort that the controller will apply on top of feedforward to steer the robot along the trajectory, not the robot's maximum velocity.
```

Comment:
Surround "in meters per second and radians per second" in parentheses or remove parentheses around "in meters and radians" for consistency.